src/amo/components/Routes/index.js

```diff
@@ -32,6 +32,10 @@ import SimulateClientError from 'core/pages/error-simulation/SimulateClientError
 import SimulateSyncError from 'core/pages/error-simulation/SimulateSyncError';
 import type { ConfigType } from 'core/types/config';
 
+const About = loadable(() =>
+  import(/* webpackPrefetch: true, webpackPreload: true, webpackChunkName: "about" */ '../../pages/StaticPages/About'),
```

Comment:
We will need some way to enforce that nothing will ever do a static import from `pages/StaticPages/About`. I found [this eslint plugin](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/dynamic-import-chunkname.md) but I think it would require us to use dynamic imports everywhere. Maybe there is a way to do it just for certain module paths?

Reply:
I am not sure I understand your comment. The first sentence and the plugin seem to be two different things to me. I interpret your first sentence as "we should not write `import ... from 'pages/StaticPages/About'` in `src/` but the ESLint plugin shows examples about the use of `webpackChunkName` (which is entirely optional here, btw)